velox/functions/lib/ArrayFunctions.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/functions/Macros.h"
+
+namespace facebook::velox::functions {
+
+/// This class implements the remove_nulls function of presto and array_compact
+/// function of spark.
```

Comment:
This is not about the semantics of this struct. Perhaps remove.

Reply:
I suggest to keep this. This comment is same as other function implement.   
https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/ArraySort.cpp#L208-L210

Reply:
@rui-mo Thanks, remove the sentence.